src/schmidt_bike_controller.py

```diff
@@ -0,0 +1,63 @@
+import sympy as sm
+import sympy.physics.control as cn
+
+
+def routh_table(polynomial, var):
+    p = sm.poly(polynomial, var)
+    n = p.degree()
+    coeffs = p.coeffs()
+    table = sm.zeros(n + 1, n + 1)
+    first = True
+    row1, row2 = [], []
+    for c in coeffs:
+        if first:
+            row1.append(c)
+            first = False
+        elif not first:
+            row2.append(c)
+            first = True
+    for i, v in enumerate(row1):
+        table[0, i] = v
+    for i, v in enumerate(row2):
+        table[1, i] = v
+    for j in range(2, n + 1):
+        for i in range(n):
+            table[j, i] = (table[j - 1, 0]*table[j - 2, i + 1] -
+                           table[j - 2, 0]*table[j - 1, i + 1])/table[j - 1, 0]
+            table[j, i] = sm.simplify(table[j, i])
+    return table
+
+
+Ib, Is, c, m, h, g, l2, l1 = sm.symbols('I_b, I_s, c, m, h, g, l2, l1',
+                                        real=True, nonnegative=True)
+KD, KP, KI, v = sm.symbols('K_D, K_P, K_I, v', real=True)
+s = sm.symbols('s')
+
+tau1_sq = (Ib + m*h**2)/m/g/h
+tau2 = l2/v
+tau3 = v/(l1 + l2)
```

Comment:
This should be the inverse (Typo in the current paper version). Also, using l instead of l1+l2 leads to much less need for manual simplification